tech-far-hub/src/components/multi-step.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from "react";
+
+import { Alert, Card, CardBody } from "@trussworks/react-uswds";
+
+export const MultiStep = ({ children }: { children: React.ReactElement }) => {
```

Comment:
I'm recognizing the pattern with these components! :)

Reply:
Phew! I need to go back to `heading` and have it pull its heading from the markdown instead of using an attribute --- hopefully I'll get time.